### PR TITLE
release 14.0.0

### PR DESCRIPTION
**Pull request by canihavethisone**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v13.4.1](https://github.com/voxpupuli/puppet-prometheus/tree/v13.4.1) (2024-03-25)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-prometheus/compare/v13.4.0...v13.4.1)
+
+**Breaking changes:**
```
Comment (bastelfreak) [inline CHANGELOG.md:11]: There are breaking changes in the release, so please make this a 14.0.0 release

Comment (canihavethisone) [reply CHANGELOG.md:11]: Done, however the PR title is the same. Cheers!

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- Add Debian 12 support [\#703](https://github.com/voxpupuli/puppet-prometheus/pull/703) ([bastelfreak](https://github.com/bastelfreak))
+- Cleanup architecture selection, dont fail on unknown architecture [\#702](https://github.com/voxpupuli/puppet-prometheus/pull/702) ([bastelfreak](https://github.com/bastelfreak))
+- mongodb\_exporter: make service name and binary path configureable [\#699](https://github.com/voxpupuli/puppet-prometheus/pull/699) ([ansgarwiechers](https://github.com/ansgarwiechers))
+- Implement wireguard exporter [\#695](https://github.com/voxpupuli/puppet-prometheus/pull/695) ([bastelfreak](https://github.com/bastelfreak))
+- Add OracleLinux support [\#687](https://github.com/voxpupuli/puppet-prometheus/pull/687) ([bastelfreak](https://github.com/bastelfreak))
+
+**Closed issues:**
+
+- mongodb\_exporter doesn't allow overriding service and binary name [\#698](https://github.com/voxpupuli/puppet-prometheus/issues/698)
+- module uses deprecated legacy function `has_keys` [\#677](https://github.com/voxpupuli/puppet-prometheus/issues/677)
```
Comment (bastelfreak) [inline CHANGELOG.md]: added skip-changelog, please regenerate the changelog

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- add option to enable tracing in Prometheus config [\#705](https://github.com/voxpupuli/puppet-prometheus/pull/705) ([fbegyn](https://github.com/fbegyn))
+- Add Debian 12 support [\#703](https://github.com/voxpupuli/puppet-prometheus/pull/703) ([bastelfreak](https://github.com/bastelfreak))
+- Cleanup architecture selection, dont fail on unknown architecture [\#702](https://github.com/voxpupuli/puppet-prometheus/pull/702) ([bastelfreak](https://github.com/bastelfreak))
+- mongodb\_exporter: make service name and binary path configureable [\#699](https://github.com/voxpupuli/puppet-prometheus/pull/699) ([ansgarwiechers](https://github.com/ansgarwiechers))
+- Implement wireguard exporter [\#695](https://github.com/voxpupuli/puppet-prometheus/pull/695) ([bastelfreak](https://github.com/bastelfreak))
+- Add OracleLinux support [\#687](https://github.com/voxpupuli/puppet-prometheus/pull/687) ([bastelfreak](https://github.com/bastelfreak))
+
+**Closed issues:**
+
+- mongodb\_exporter doesn't allow overriding service and binary name [\#698](https://github.com/voxpupuli/puppet-prometheus/issues/698)
```
Comment (bastelfreak) [inline CHANGELOG.md]: same here

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**